### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by ttthl** [xw-hu/FSDNet#6]
Hello, 

firstly thanks for the work. 

I have a question for the loss function. Here you use the L1 loss instead of cross entropy loss. Is there any specific reason for this? From my side, shadow detection is more like a pixel-wise classification task rather than regression task. And I've also checked your previous DSC paper, there you used the weighted BCE loss. So just a little bit curious why you changed it to L1 loss.

**Comment by xw-hu** [comment]
Thanks for your question. There is no specific reason for these two different losses. From my experience, the L1 loss is more suitable for the backbone network used in this paper. I just tried different losses and chose the better one.

**Comment by ttthl** [comment]
ok got it. Thanks for your reply!
